packages/workbox-routing/_default.mjs

```diff
@@ -14,7 +14,12 @@
   limitations under the License.
 */
 
-import {assert, WorkboxError, cacheNames} from 'workbox-core/_private.mjs';
+import {
+  WorkboxError,
+  assert,
+  cacheNames,
+  logger,
+} from 'workbox-core/_private.mjs';
```

Comment:
This will need to be changed to:

```javascript
import {WorkboxError} from 'workbox-core/_private/WorkboxError.mjs';
import {assert} from 'workbox-core/_private/assert.mjs';
...
```